source/image-handler/test/thumbor-mapper/filter.spec.ts

```diff
@@ -638,4 +638,68 @@ describe("filter", () => {
     };
     expect(edits).toEqual(expectedResult.edits);
   });
+
+  it("Should pass for fit-in combined with watermark in folder", () => {
+    // watermark params: bucket, key, xPos, yPos, alpha, wRatio, hRatio
+    const path = "/fit-in/400x400/filters:watermark(bucket,folder/key.png,0,0)/image.jpg";
+
+    // Act
+    const thumborMapper = new ThumborMapper();
+    const edits = thumborMapper.mapPathToEdits(path);
+
+    // Assert
+    const expectedResult = {
+      edits: {
+        resize: {
+          width: 400,
+          height: 400,
+          fit: "inside",
+        },
+        overlayWith: {
+          bucket: "bucket",
+          key: "folder/key.png",
+          alpha: undefined,
+          wRatio: undefined,
+          hRatio: undefined,
+          options: {
+            left: "0",
+            top: "0",
+          },
+        },
+      },
+    };
+    expect(edits).toEqual(expectedResult.edits);
+  });
+
+  it("Should pass for fit-in combined with watermark not in folder", () => {
```

Comment:
does "should pass" here mean that the test should be green or that the code should "pass" in this circumstance?

I personally shy away from adding any extra English prefixes like "should" to the test name in favor of trying to succinctly describe the exact input -> result. Something like "fit-in with watermark returns expected result"

Reply:
Followed the pattern of all of the existing unit tests in using the "Should pass" prefix.